<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>771</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="zoomSlider">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>160</y>
      <width>21</width>
      <height>246</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>255</width>
      <height>599</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="Next_stop_title">
       <property name="text">
        <string>Next stop: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Speed_title">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0" colspan="2">
      <widget class="QCheckBox" name="Show_congestion_check">
       <property name="text">
        <string>Show congestion</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QComboBox" name="congestionChoiceBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="Line_selector">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="Last_stop_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="Vehicle_number_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="Street_selector">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="Position_line_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="Start_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Line_title">
       <property name="text">
        <string>Line:</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="Position_stop_title">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="Next_stop_arrival_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Position_line_title">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="Next_stop_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="Next_stop_arrival_title">
       <property name="text">
        <string>Next stop arrival:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="End_title">
       <property name="text">
        <string>End: </string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QComboBox" name="Stop_selector">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QPushButton" name="editRouteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit route</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Vehicle_number_title">
       <property name="text">
        <string>Vehicle number:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Last_stop_title">
       <property name="text">
        <string>Last stop: </string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="Show_route_selected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show route</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="Congestion_title">
       <property name="text">
        <string>Congestion:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="cancelEditButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0" colspan="2">
      <widget class="QCheckBox" name="Show_street_names_check">
       <property name="text">
        <string>Show street names</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="Position_stop_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="Start_title">
       <property name="text">
        <string>Start:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Time_title">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="End_selected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Time_counter">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="Street_title">
       <property name="text">
        <string>Street:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Speed_selector">
       <item>
        <property name="text">
         <string>1x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100x</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="Stop_title">
       <property name="text">
        <string>Stop:</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="saveEditButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="2">
      <widget class="QPushButton" name="timetableButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Timetable</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuICP_projekt">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_help"/>
   </widget>
   <addaction name="menuICP_projekt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_help">
   <property name="text">
    <string>Show help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
